### PR TITLE
Add support for pipeline when disabled etc

### PR DESCRIPTION
**Pull request by tomviner**
Changes:
- Add support for pipeline when disabled
- Fix test requirement pinning
- Drop support for the oldest Nameko (2.6)
- Add tests for Python 3.8

### CODE REVIEW

File: test/test_lazy_client.py [hunk 1]
```diff
@@ -142,7 +142,7 @@ def stats_config(self, request, stats_config):
         return stats_config['STATSD'][request.param].copy()
 
     @pytest.mark.parametrize('method', [
-        'incr', 'decr', 'gauge', 'set', 'timing'
+        'incr', 'decr', 'gauge', 'set', 'timing', 'pipeline',
```
Comment (heindsight) [inline test/test_lazy_client.py]: Maybe we should add `timer` here too.

File: CHANGELOG.md [hunk 2]
```diff
@@ -13,8 +13,10 @@ Version 0.1.1
 
 * Deprecate the `ServiceBaseMeta` metaclass and the `name` argument to the
   `StatsD` dependency provider.
+* Add support for pipeline when disabled
 * Test on older Nameko versions and against pre-release versions of
   dependencies.
+* Test on Python 3.8
```
Comment (heindsight) [inline CHANGELOG.md]: Also mention dropping support for Nameko 2.6

Comment (tomviner) [reply CHANGELOG.md]: Support for 2.6 was not released yet. It's in the list of changes on master but not released https://github.com/sohonetlabs/nameko-statsd/compare/0.1.0...HEAD

But I'll clarify the text to mention 2.7 as the oldest supported.

File: test/test_lazy_client.py (hunk 1 above)
Comment (tomviner) [reply test/test_lazy_client.py]: Fixed

File: CHANGELOG.md (hunk 2 above)
Comment (heindsight) [reply CHANGELOG.md]: 👌

**Pull request merged**